src/home/banner/dto/createBannerRequest.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty } from 'class-validator';
+
+export class createBannerRequestDto {
```

Comment:
```suggestion
export class CreateBannerRequestDto {
```

클래스니 대문자로 하는건 어떨지...